### PR TITLE
modify sequence_conv_xpu op test. test=kunlun

### PR DESCRIPTION
**Pull request by tiancaitzp**
### PR types

Others 

### PR changes

Others 

### Describe

根据内部规范修改xpu op test

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py [hunk 1]
```diff
@@ -73,188 +75,198 @@ def seqconv(x,
...
+                    and self.padding_trainable:
+                print("If context_start is 0 " \
+                      "and context_length is 1," \
+                      " padding_trainable should be false.")
+                return
+
+            # one level, batch size
+            x = np.random.uniform(-6.10907e-05, 0.000104218,
+                                  [self.input_size[0],
+                                   self.input_size[1]]).astype('float32')
```
Comment (XiaociZhang) [inline python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py]: 用self.dtype

File: python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py [hunk 2]
```diff
@@ -73,188 +75,198 @@ def seqconv(x,
...
+            w = np.random.uniform(-3.17068e-05, 0.000159822, [
+                self.context_length * self.input_size[1],
+                self.output_represention
+            ]).astype('float32')
```
Comment (XiaociZhang) [inline python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py]: 同上

File: python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py [hunk 3]
```diff
@@ -73,188 +75,198 @@ def seqconv(x,
...
+
+            begin_pad = np.max([0, -self.context_start])
+            end_pad = np.max([0, self.context_start + self.context_length - 1])
+            total_pad = begin_pad + end_pad
+            padding_data = np.random.uniform(
+                0, 0, [total_pad, self.input_size[1]]).astype('float32')
```
Comment (XiaociZhang) [inline python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py]: 同上

**Review by XiaociZhang — CHANGES_REQUESTED** [review]

File: python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py (hunk 3 above)
Comment (tiancaitzp) [reply python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py]: done

File: python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py (hunk 2 above)
Comment (tiancaitzp) [reply python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py]: done

File: python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py (hunk 1 above)
Comment (tiancaitzp) [reply python/paddle/fluid/tests/unittests/xpu/test_sequence_conv_op_xpu.py]: done

**Review by XiaociZhang — APPROVED** [review]
LGTM

**Review by QingshuChen — APPROVED** [review]

**Pull request merged**